From 70e4e8901a6514965ea0a3f7a7d8019a346c2fed Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:41:23 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(fbe605c9f)..lineage-16.0-android-9.0.0_r31(2eb597cd5) (DO
 NOT SUBMIT)

Android Build Merger (Role) (5):
 - [automerger] Patch URI vulnerability in contact photo editing am:
   ccfd94b965
 - [automerger] Patch URI vulnerability in contact photo editing am:
   ccfd94b965 am: 2fc4d78bfc
 - [automerger] Patch URI vulnerability in contact photo editing am:
   ccfd94b965 am: 2fc4d78bfc am: fc6e94648d
 - [automerger] Patch URI vulnerability in contact photo editing am:
   ccfd94b965 am: 2fc4d78bfc am: fc6e94648d am: 3ffe76ed9c
 - [automerger] Patch URI vulnerability in contact photo editing am:
   ccfd94b965 am: 2fc4d78bfc am: fc6e94648d am: 3ffe76ed9c am:
   456f745849

Bill Yi (5):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE

Gary Mai (4):
 - Patch URI vulnerability in contact photo editing
 - [automerger] Patch URI vulnerability in contact photo editing am:
   ccfd94b965 am: 2fc4d78bfc am: fc6e94648d am: 3ffe76ed9c am:
   456f745849 am: 9df22bbe22
 - [automerger] Patch URI vulnerability in contact photo editing am:
   ccfd94b965 am: 2fc4d78bfc am: fc6e94648d am: 3ffe76ed9c am:
   456f745849 am: 9df22bbe22 am: b96186029f
 - [automerger] Patch URI vulnerability in contact photo editing am:
   ccfd94b965 am: 2fc4d78bfc am: fc6e94648d am: 3ffe76ed9c am:
   456f745849 am: 9df22bbe22 am: b96186029f am: 655ad64631

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (5):
 - Snap for 5044688 from 75f11f429a9a039b8b41066828b72b77c97b5236 to
   pi-qpr2-release
 - Snap for 5073521 from a9aca04c647757bca9e4d008e240a964cea8c67d to
   pi-qpr2-release
 - Snap for 5101444 from acdfb82c89a58e0493f786e074862dbfda93399a to
   pi-qpr2-release
 - Snap for 5115288 from 8853bf6ef9136e13015e439f93cf93ced7da6a8e to
   pi-qpr2-release
 - Snap for 5130385 from 01a0bdc06af8833bd0fdd40e04fb81f0d92c4beb to
   pi-qpr2-release

Change-Id: I01399808955f1f794985c8b25f7424166601a805

Change-Id: Ic5efeba86cd1b3af25c2a4b752b770f88968399a
---
 res/values-bn/strings.xml     |  2 +-
 res/values-bs/strings.xml     | 18 +++++++++---------
 res/values-ca/strings.xml     |  2 +-
 res/values-da/strings.xml     | 14 +++++++-------
 res/values-es-rUS/strings.xml |  4 ++--
 res/values-eu/strings.xml     |  2 +-
 res/values-fr/strings.xml     |  2 +-
 res/values-it/strings.xml     |  2 +-
 res/values-km/strings.xml     |  2 +-
 res/values-ky/strings.xml     |  2 +-
 res/values-mr/strings.xml     | 36 +++++++++++++++++------------------
 res/values-my/strings.xml     |  2 +-
 res/values-nl/strings.xml     | 10 +++++-----
 res/values-pl/strings.xml     |  2 +-
 res/values-pt-rBR/strings.xml | 18 +++++++++---------
 res/values-pt-rPT/strings.xml |  2 +-
 res/values-pt/strings.xml     | 18 +++++++++---------
 res/values-ru/strings.xml     |  2 +-
 res/values-th/strings.xml     |  2 +-
 res/values-uz/strings.xml     |  2 +-
 res/values-zh-rCN/strings.xml |  2 +-
 res/values-zh-rHK/strings.xml |  2 +-
 22 files changed, 74 insertions(+), 74 deletions(-)

diff --git a/res/values-bn/strings.xml b/res/values-bn/strings.xml
index b0e63c751..988a4e404 100644
--- a/res/values-bn/strings.xml
+++ b/res/values-bn/strings.xml
@@ -489,7 +489,7 @@
     <string name="call_subject_type_and_number" msgid="7667188212129152558">"<xliff:g id="TYPE">%1$s</xliff:g> • <xliff:g id="NUMBER">%2$s</xliff:g>"</string>
     <string name="about_build_version" msgid="1765533099416999801">"বিল্ডের সংস্করণ"</string>
     <string name="about_open_source_licenses" msgid="6479990452352919641">"ওপেন সোর্স লাইসেন্স"</string>
-    <string name="about_open_source_licenses_summary" msgid="57418386931763994">"মুক্ত উৎস সফ্টওয়্যারের লাইসেন্স বিবরণ"</string>
+    <string name="about_open_source_licenses_summary" msgid="57418386931763994">"ওপেন সোর্স সফ্টওয়্যারের লাইসেন্স বিবরণ"</string>
     <string name="about_privacy_policy" msgid="3705518622499152626">"গোপনীয়তা নীতি"</string>
     <string name="about_terms_of_service" msgid="4642400812150296723">"পরিষেবার শর্তাবলী"</string>
     <string name="activity_title_licenses" msgid="5467767062737708066">"ওপেন সোর্স লাইসেন্স"</string>
diff --git a/res/values-bs/strings.xml b/res/values-bs/strings.xml
index b62ef50af..39d6c3f6e 100644
--- a/res/values-bs/strings.xml
+++ b/res/values-bs/strings.xml
@@ -26,7 +26,7 @@
     <string name="contactInsertOrEditActivityTitle" msgid="6527505405325862674">"Dodaj u kontakt"</string>
     <string name="contactPickerActivityTitle" msgid="4301062192337417640">"Odabir kontakta"</string>
     <string name="groupMemberPickerActivityTitle" msgid="1431750793695262522">"Odaberi"</string>
-    <string name="header_entry_contact_list_adapter_header_title" msgid="2436981165830115659">"Izrada novog kontakta"</string>
+    <string name="header_entry_contact_list_adapter_header_title" msgid="2436981165830115659">"Kreiraj novi kontakt"</string>
     <string name="searchHint" msgid="8482945356247760701">"Pretraži kontakte"</string>
     <string name="menu_addStar" msgid="2908478235715404876">"Dodaj u omiljeno"</string>
     <string name="menu_removeStar" msgid="5844227078364227030">"Ukloni iz omiljenih"</string>
@@ -109,11 +109,11 @@
     <string name="contactSavedNamedToast" msgid="895136068894549611">"Kontakt <xliff:g id="DISPLAY_NAME">%s</xliff:g> je sačuvan"</string>
     <string name="contactSavedToast" msgid="9171862279493213075">"Kontakt je sačuvan"</string>
     <string name="contactUnlinkedToast" msgid="7122823195786012553">"Kontakti su razdvojeni"</string>
-    <string name="contactSavedErrorToast" msgid="4827033473908688031">"Nije moguće sačuvati promjene kontakta"</string>
+    <string name="contactSavedErrorToast" msgid="4827033473908688031">"Pohranjivanje promjena kontakta nije uspjelo"</string>
     <string name="contactUnlinkErrorToast" msgid="2706297508166952431">"Nije moguće razdvojiti kontakt"</string>
     <string name="contactJoinErrorToast" msgid="5735129234573327701">"Nije moguće povezati kontakt"</string>
     <string name="contactGenericErrorToast" msgid="3885457515665783976">"Greška prilikom spremanja kontakta"</string>
-    <string name="contactPhotoSavedErrorToast" msgid="6860883564984042194">"Nije moguće sačuvati promjene fotografije kontakta"</string>
+    <string name="contactPhotoSavedErrorToast" msgid="6860883564984042194">"Pohranjivanje promjena fotografije kontakta nije uspjelo"</string>
     <string name="groupLoadErrorToast" msgid="7536267148196064554">"Učitavanje oznake nije uspjelo"</string>
     <string name="groupDeletedToast" msgid="520896687873262027">"Oznaka je izbrisana"</string>
     <string name="groupCreatedToast" msgid="1924195126172834870">"Oznaka je napravljena"</string>
@@ -121,7 +121,7 @@
     <string name="groupUpdatedToast" msgid="3667977658676267687">"Oznaka je ažurirana"</string>
     <string name="groupMembersRemovedToast" msgid="3510563559799376603">"Uklonjeno iz oznake"</string>
     <string name="groupMembersAddedToast" msgid="4824834898718972768">"Dodano oznaci"</string>
-    <string name="groupSavedErrorToast" msgid="2355891714292740162">"Nije moguće sačuvati promjene oznake"</string>
+    <string name="groupSavedErrorToast" msgid="2355891714292740162">"Pohranjivanje promjena oznake nije uspjelo"</string>
     <string name="groupExistsErrorMessage" msgid="5196811283836946189">"Ta oznaka već postoji"</string>
     <string name="groupSomeContactsNoEmailsToast" msgid="8211161470235548732">"Neki kontakti nemaju adrese e-pošte."</string>
     <string name="groupSomeContactsNoPhonesToast" msgid="5530760834827530509">"Neki kontakti nemaju brojeve telefona."</string>
@@ -193,7 +193,7 @@
     <string name="contact_editor_prompt_zero_accounts" msgid="6648376557574360096">"Odvojite malo vremena da dodate račun putem kojeg će se kontakti sigurnosno kopirati na Google."</string>
     <string name="contact_editor_prompt_one_account" msgid="3087691056345099310">"Novi kontakti će biti sačuvani na račun <xliff:g id="ACCOUNT_NAME">%1$s</xliff:g>."</string>
     <string name="contact_editor_prompt_multiple_accounts" msgid="8565761674283473549">"Odaberite zadani račun za nove kontakte:"</string>
-    <string name="contact_editor_title_new_contact" msgid="3042788718983070111">"Izrada novog kontakta"</string>
+    <string name="contact_editor_title_new_contact" msgid="3042788718983070111">"Kreiraj novi kontakt"</string>
     <string name="contact_editor_title_existing_contact" msgid="4898475703683187798">"Uređivanje kontakta"</string>
     <string name="contact_editor_title_read_only_contact" msgid="2443496574528842237">"Samo prikaz"</string>
     <string name="contact_editor_pick_raw_contact_to_edit_dialog_title" msgid="4186699385645902152">"Odaberite kontakt za uređivanje"</string>
@@ -307,7 +307,7 @@
     <string name="sms_by_shortcut" msgid="7741770672976099517">"<xliff:g id="CONTACT_NAME">%s</xliff:g> (Poruka)"</string>
     <string name="description_video_call" msgid="7120921378651700947">"Uputi video poziv"</string>
     <string name="clearFrequentsConfirmation_title" msgid="766292372438450432">"Obrisati često kontaktirane osobe?"</string>
-    <string name="clearFrequentsConfirmation" msgid="2270554975938265734">"Obrisat ćete listu često kontaktiranih osoba u aplikaciji Kontakti i Aplikaciji za telefon, pa će aplikacije za e-poštu morati ispočetka učiti vaše postavke adresa."</string>
+    <string name="clearFrequentsConfirmation" msgid="2270554975938265734">"Obrisat ćete listu često kontaktiranih osoba u aplikacijama Kontakti i Telefon, pa će aplikacije za e-poštu morati ispočetka učiti vaše postavke adresa."</string>
     <string name="clearFrequentsProgress_title" msgid="5157001637482794212">"Brisanje često kontaktiranih osoba…"</string>
     <string name="status_available" msgid="5586870015822828392">"Dostupno"</string>
     <string name="status_away" msgid="1838861100379804730">"Odsutan"</string>
@@ -505,9 +505,9 @@
     <string name="description_search_video_call" msgid="5841525580339803272">"Uputi video poziv"</string>
     <string name="description_delete_contact" msgid="53835657343783663">"Izbriši"</string>
     <string name="description_no_name_header" msgid="8884991311595943271">"Elipsa"</string>
-    <string name="callDurationSecondFormat" msgid="7067644915903528776">"<xliff:g id="SECONDS">%s</xliff:g> sek."</string>
-    <string name="callDurationMinuteFormat" msgid="4647095486747447674">"<xliff:g id="MINUTES">%1$s</xliff:g> min. <xliff:g id="SECONDS">%2$s</xliff:g> sek."</string>
-    <string name="callDurationHourFormat" msgid="7392254193808506640">"<xliff:g id="MINUTES_0">%1$s</xliff:g> h <xliff:g id="MINUTES_1">%2$s</xliff:g> min. <xliff:g id="SECONDS">%3$s</xliff:g> sek."</string>
+    <string name="callDurationSecondFormat" msgid="7067644915903528776">"<xliff:g id="SECONDS">%s</xliff:g> s"</string>
+    <string name="callDurationMinuteFormat" msgid="4647095486747447674">"<xliff:g id="MINUTES">%1$s</xliff:g> min. <xliff:g id="SECONDS">%2$s</xliff:g> s"</string>
+    <string name="callDurationHourFormat" msgid="7392254193808506640">"<xliff:g id="MINUTES_0">%1$s</xliff:g> h <xliff:g id="MINUTES_1">%2$s</xliff:g> min. <xliff:g id="SECONDS">%3$s</xliff:g> s"</string>
     <string name="dynamic_shortcut_disabled_message" msgid="249939425761315252">"Ova prečica je onemogućena"</string>
     <string name="dynamic_shortcut_contact_removed_message" msgid="6845645831837615899">"Kontakt je uklonjen"</string>
     <string name="sim_import_button_text" msgid="4270841592496619317">"Uvezi"</string>
diff --git a/res/values-ca/strings.xml b/res/values-ca/strings.xml
index 9d91b9ed3..f6d872672 100644
--- a/res/values-ca/strings.xml
+++ b/res/values-ca/strings.xml
@@ -329,7 +329,7 @@
     <string name="name_suffix" msgid="3855278445375651441">"Sufix del nom"</string>
     <string name="name_phonetic" msgid="4259595234312430484">"Nom fonètic"</string>
     <string name="name_phonetic_given" msgid="8723179018384187631">"Nom fonètic"</string>
-    <string name="name_phonetic_middle" msgid="8643721493320405200">"Primer cognom fonètic"</string>
+    <string name="name_phonetic_middle" msgid="8643721493320405200">"Segon nom fonètic"</string>
     <string name="name_phonetic_family" msgid="2640133663656011626">"Cognoms fonètics"</string>
     <string name="phoneLabelsGroup" msgid="6468091477851199285">"Telèfon"</string>
     <string name="emailLabelsGroup" msgid="8389931313045344406">"Correu electrònic"</string>
diff --git a/res/values-da/strings.xml b/res/values-da/strings.xml
index 0d8b2ecef..8bb208996 100644
--- a/res/values-da/strings.xml
+++ b/res/values-da/strings.xml
@@ -16,8 +16,8 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="applicationLabel" msgid="3906689777043645443">"Kontaktpersoner"</string>
-    <string name="contactsList" msgid="8661624236494819731">"Kontaktpersoner"</string>
+    <string name="applicationLabel" msgid="3906689777043645443">"Kontakter"</string>
+    <string name="contactsList" msgid="8661624236494819731">"Kontakter"</string>
     <string name="shortcut_add_contact" msgid="7476283631356909204">"Tilføj kontakt"</string>
     <string name="shortcutContact" msgid="749243779392912958">"Kontakt"</string>
     <string name="shortcutDialContact" msgid="746622101599186779">"Direkte opkald"</string>
@@ -87,7 +87,7 @@
     <string name="readOnlyContactDeleteConfirmation" msgid="4717023505914239809">"Data om denne kontakt fra skrivebeskyttede konti bliver skjult, men ikke slettet."</string>
     <string name="single_delete_confirmation" msgid="3106905986948679720">"Vil du slette denne kontakt?"</string>
     <string name="batch_delete_confirmation" msgid="3984346060324014108">"Vil du slette de valgte kontaktpersoner?"</string>
-    <string name="batch_delete_read_only_contact_confirmation" msgid="8411117621035639964">"Kontaktpersoner fra dine skrivebeskyttede konti kan ikke slettes, men de kan skjules."</string>
+    <string name="batch_delete_read_only_contact_confirmation" msgid="8411117621035639964">"Kontakter fra dine skrivebeskyttede konti kan ikke slettes, men de kan skjules."</string>
     <string name="batch_delete_multiple_accounts_confirmation" msgid="8207205649127030030">"De kontaktpersoner, du vil slette, indeholder oplysninger fra flere konti. Oplysninger fra skrivebeskyttede konti skjules fremfor at blive slettet."</string>
     <string name="multipleContactDeleteConfirmation" msgid="5235324124905653550">"Sletning af denne kontakt sletter oplysninger fra flere konti."</string>
     <string name="deleteConfirmation" msgid="3512271779086656043">"Vil du slette denne kontakt?"</string>
@@ -303,7 +303,7 @@
     <string name="sms_by_shortcut" msgid="7741770672976099517">"<xliff:g id="CONTACT_NAME">%s</xliff:g> (send besked)"</string>
     <string name="description_video_call" msgid="7120921378651700947">"Foretag videoopkald"</string>
     <string name="clearFrequentsConfirmation_title" msgid="766292372438450432">"Vil du rydde de ofte kontaktede?"</string>
-    <string name="clearFrequentsConfirmation" msgid="2270554975938265734">"Hvis du gør dette, rydder du listen over personer, som du ofte kontakter, i dine apps Kontaktpersoner og Telefon. Du vil samtidig tvinge e-mailapps til at lære dine adressepræferencer fra bunden."</string>
+    <string name="clearFrequentsConfirmation" msgid="2270554975938265734">"Hvis du gør dette, rydder du listen over personer, som du ofte kontakter, i dine apps Kontakter og Telefon. Du vil samtidig tvinge e-mailapps til at lære dine adressepræferencer fra bunden."</string>
     <string name="clearFrequentsProgress_title" msgid="5157001637482794212">"Rydder ofte kontaktede personer…"</string>
     <string name="status_available" msgid="5586870015822828392">"Tilgængelig"</string>
     <string name="status_away" msgid="1838861100379804730">"Ikke til stede"</string>
@@ -316,7 +316,7 @@
     <string name="missing_name" msgid="8745511583852904385">"(Intet navn)"</string>
     <string name="favoritesFrequentContacted" msgid="6184232487472425690">"Jævnligt kontaktet"</string>
     <string name="list_filter_phones" msgid="735313795643493365">"Alle kontaktpersoner med telefonnumre"</string>
-    <string name="list_filter_phones_work" msgid="1470173699551475015">"Kontaktpersoner for arbejdsprofil"</string>
+    <string name="list_filter_phones_work" msgid="1470173699551475015">"Kontakter for arbejdsprofil"</string>
     <string name="view_updates_from_group" msgid="1782685984905600034">"Se opdateringer"</string>
     <string name="account_phone" msgid="7128032778471187553">"Enhed"</string>
     <string name="account_sim" msgid="6519016427905087952">"SIM-kort"</string>
@@ -459,7 +459,7 @@
     <string name="default_editor_account" msgid="699591683362420991">"Standardkonto til nye kontaktpersoner"</string>
     <string name="settings_my_info_title" msgid="1534272456405343119">"Mine oplysninger"</string>
     <string name="set_up_profile" msgid="7370213843590143771">"Konfigurer din profil"</string>
-    <string name="setting_about" msgid="7014388749752042863">"Om Kontaktpersoner"</string>
+    <string name="setting_about" msgid="7014388749752042863">"Om Kontakter"</string>
     <string name="share_favorite_contacts" msgid="4280926751003081042">"Del foretrukne kontaktpersoner"</string>
     <string name="share_contacts" msgid="8109287987498711664">"Del alle kontaktpersoner"</string>
     <string name="share_contacts_failure" msgid="1216431977330560559">"Kontaktpersonerne kunne ikke deles."</string>
@@ -519,7 +519,7 @@
     <string name="dismiss_sync_alert" msgid="8941131776605093596">"Luk"</string>
     <string name="account_sync_off" msgid="7646143254066243924">"Kontosynkronisering er slået fra. Tryk for at slå funktionen til."</string>
     <string name="turn_auto_sync_on_dialog_title" msgid="333965456829670541">"Vil du slå automatisk synkronisering til?"</string>
-    <string name="turn_auto_sync_on_dialog_body" msgid="5025336917533693997">"De ændringer, du foretager af alle apps og konti og ikke kun Google Kontaktpersoner, holdes opdateret mellem nettet og dine enheder."</string>
+    <string name="turn_auto_sync_on_dialog_body" msgid="5025336917533693997">"De ændringer, du foretager af alle apps og konti og ikke kun Google Kontakter, holdes opdateret mellem nettet og dine enheder."</string>
     <string name="turn_auto_sync_on_dialog_confirm_btn" msgid="1070332014838295599">"Slå til"</string>
     <string name="connection_error_message" msgid="7046408921529247089">"Der er ingen forbindelse"</string>
     <string name="single_sim_display_label" msgid="6985770499439371497">"SIM-kort"</string>
diff --git a/res/values-es-rUS/strings.xml b/res/values-es-rUS/strings.xml
index eaad2c4f7..1a7ff089b 100644
--- a/res/values-es-rUS/strings.xml
+++ b/res/values-es-rUS/strings.xml
@@ -434,7 +434,7 @@
     <string name="reading_vcard_failed_title" msgid="4251647443358422855">"No se pudieron leer los datos de vCard."</string>
     <string name="importing_vcard_finished_title" msgid="3341541727268747967">"Finalizó la importación de la vCard <xliff:g id="FILENAME">%s</xliff:g>"</string>
     <string name="importing_vcard_canceled_title" msgid="2147475978165599336">"Se canceló la importación de <xliff:g id="FILENAME">%s</xliff:g>."</string>
-    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"<xliff:g id="FILENAME">%s</xliff:g> se importará en breve."</string>
+    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"Se importará(n) en breve <xliff:g id="FILENAME">%s</xliff:g>"</string>
     <string name="vcard_import_will_start_message_with_default_name" msgid="1022969530654129470">"El archivo se importará en breve."</string>
     <string name="vcard_import_request_rejected_message" msgid="2890471184508516011">"Se rechazó la solicitud de importación de la vCard. Vuelve a intentarlo más tarde."</string>
     <string name="contacts_export_will_start_message" msgid="8538705791417534431">"Los contactos se exportarán en breve."</string>
@@ -490,7 +490,7 @@
     <string name="about_build_version" msgid="1765533099416999801">"Versión de compilación"</string>
     <string name="about_open_source_licenses" msgid="6479990452352919641">"Licencias de código abierto"</string>
     <string name="about_open_source_licenses_summary" msgid="57418386931763994">"Detalles de la licencia de software de código abierto"</string>
-    <string name="about_privacy_policy" msgid="3705518622499152626">"Política de privacidad"</string>
+    <string name="about_privacy_policy" msgid="3705518622499152626">"Política de Privacidad"</string>
     <string name="about_terms_of_service" msgid="4642400812150296723">"Condiciones del servicio"</string>
     <string name="activity_title_licenses" msgid="5467767062737708066">"Licencias de código abierto"</string>
     <string name="url_open_error_toast" msgid="452592089815420457">"No se pudo abrir la URL."</string>
diff --git a/res/values-eu/strings.xml b/res/values-eu/strings.xml
index 0bfe8412b..31aa80de0 100644
--- a/res/values-eu/strings.xml
+++ b/res/values-eu/strings.xml
@@ -487,7 +487,7 @@
     <string name="send_and_call_button" msgid="7740295432834590737">"BIDALI ETA DEITU"</string>
     <string name="call_subject_limit" msgid="4545212901205397669">"<xliff:g id="COUNT">%1$s</xliff:g>/<xliff:g id="LIMIT">%2$s</xliff:g>"</string>
     <string name="call_subject_type_and_number" msgid="7667188212129152558">"<xliff:g id="TYPE">%1$s</xliff:g> • <xliff:g id="NUMBER">%2$s</xliff:g>"</string>
-    <string name="about_build_version" msgid="1765533099416999801">"Konpilazioaren bertsioa"</string>
+    <string name="about_build_version" msgid="1765533099416999801">"Konpilazio-bertsioa"</string>
     <string name="about_open_source_licenses" msgid="6479990452352919641">"Kode irekiko lizentziak"</string>
     <string name="about_open_source_licenses_summary" msgid="57418386931763994">"Kode irekiko softwarearen lizentziaren xehetasunak"</string>
     <string name="about_privacy_policy" msgid="3705518622499152626">"Pribatutasun-gidalerroak"</string>
diff --git a/res/values-fr/strings.xml b/res/values-fr/strings.xml
index f9d6a286f..d86d3c07c 100644
--- a/res/values-fr/strings.xml
+++ b/res/values-fr/strings.xml
@@ -434,7 +434,7 @@
     <string name="reading_vcard_failed_title" msgid="4251647443358422855">"Impossible de lire les données vCard."</string>
     <string name="importing_vcard_finished_title" msgid="3341541727268747967">"Fichier vCard <xliff:g id="FILENAME">%s</xliff:g> importé"</string>
     <string name="importing_vcard_canceled_title" msgid="2147475978165599336">"Importation de \"<xliff:g id="FILENAME">%s</xliff:g>\" annulée"</string>
-    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"Importation du fichier <xliff:g id="FILENAME">%s</xliff:g> imminente"</string>
+    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"Importation imminente : <xliff:g id="FILENAME">%s</xliff:g>"</string>
     <string name="vcard_import_will_start_message_with_default_name" msgid="1022969530654129470">"Le fichier va bientôt être importé."</string>
     <string name="vcard_import_request_rejected_message" msgid="2890471184508516011">"La demande d\'importation du fichier vCard a été rejetée. Veuillez réessayer ultérieurement."</string>
     <string name="contacts_export_will_start_message" msgid="8538705791417534431">"Les contacts seront bientôt exportés."</string>
diff --git a/res/values-it/strings.xml b/res/values-it/strings.xml
index 6b743d5ba..216fb38d2 100644
--- a/res/values-it/strings.xml
+++ b/res/values-it/strings.xml
@@ -434,7 +434,7 @@
     <string name="reading_vcard_failed_title" msgid="4251647443358422855">"Lettura dati vCard non riuscita"</string>
     <string name="importing_vcard_finished_title" msgid="3341541727268747967">"Importazione vCard <xliff:g id="FILENAME">%s</xliff:g> terminata"</string>
     <string name="importing_vcard_canceled_title" msgid="2147475978165599336">"Importazione di <xliff:g id="FILENAME">%s</xliff:g> annullata"</string>
-    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"La vCard <xliff:g id="FILENAME">%s</xliff:g> verrà importata a breve."</string>
+    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"Contatti che verranno importati a breve: <xliff:g id="FILENAME">%s</xliff:g>"</string>
     <string name="vcard_import_will_start_message_with_default_name" msgid="1022969530654129470">"Il file sarà importato a breve."</string>
     <string name="vcard_import_request_rejected_message" msgid="2890471184508516011">"Richiesta importazione vCard rifiutata. Riprova più tardi."</string>
     <string name="contacts_export_will_start_message" msgid="8538705791417534431">"I contatti verranno esportati a breve."</string>
diff --git a/res/values-km/strings.xml b/res/values-km/strings.xml
index 4425ce3f4..0d975a45f 100644
--- a/res/values-km/strings.xml
+++ b/res/values-km/strings.xml
@@ -475,7 +475,7 @@
     <string name="contactsFavoritesLabel" msgid="8417039765586853670">"សំណព្វ"</string>
     <string name="menu_import" msgid="6107961135813836467">"នាំចូល"</string>
     <string name="menu_export" msgid="2658783911863503902">"នាំ​ចេញ"</string>
-    <string name="menu_blocked_numbers" msgid="5272951629083025995">"លេខបានរារាំង"</string>
+    <string name="menu_blocked_numbers" msgid="5272951629083025995">"លេខដែលបានទប់ស្កាត់"</string>
     <string name="contact_status_update_attribution" msgid="752179367353018597">"តាម​រយៈ <xliff:g id="SOURCE">%1$s</xliff:g>"</string>
     <string name="contact_status_update_attribution_with_date" msgid="7358045508107825068">"<xliff:g id="DATE">%1$s</xliff:g> តាម​រយៈ <xliff:g id="SOURCE">%2$s</xliff:g>"</string>
     <string name="action_menu_back_from_search" msgid="8793348588949233348">"បញ្ឈប់​ការ​ស្វែងរក"</string>
diff --git a/res/values-ky/strings.xml b/res/values-ky/strings.xml
index d728d4a4f..c64e36972 100644
--- a/res/values-ky/strings.xml
+++ b/res/values-ky/strings.xml
@@ -390,7 +390,7 @@
     <string name="dialog_sync_add" msgid="8267045393119375803">"Шайкештирүү тобуна кошуу"</string>
     <string name="display_more_groups" msgid="2682547080423434170">"Дагы топтор…"</string>
     <string name="display_warn_remove_ungrouped" msgid="8872290721676651414">"\"<xliff:g id="GROUP">%s</xliff:g>\" тобун шайкештирүүдөн алып салуу топко кирбеген башка байланыштарды дагы шайкештирүүдөн чыгарып салат."</string>
-    <string name="savingDisplayGroups" msgid="2133152192716475939">"Көрсөтүү параметрлери сакталууда…"</string>
+    <string name="savingDisplayGroups" msgid="2133152192716475939">"Жөндөөлөр сакталууда…"</string>
     <string name="listCustomView" msgid="1840624396582117590">"Ылайыкташтырылган көрүнүш"</string>
     <string name="dialog_new_contact_account" msgid="4969619718062454756">"Өткөрүлүп алынган байланыштар төмөнкүгө сакталсын:"</string>
     <string name="import_from_sim" msgid="6912071635295799131">"SIM-карта"</string>
diff --git a/res/values-mr/strings.xml b/res/values-mr/strings.xml
index 1e39aad59..8f56f937c 100644
--- a/res/values-mr/strings.xml
+++ b/res/values-mr/strings.xml
@@ -47,9 +47,9 @@
     <string name="menu_new_group_action_bar" msgid="3127059073617415221">"लेबल तयार करा"</string>
     <string name="splitConfirmation" msgid="7342030840130187290">"एकाधिक संपर्कांमध्‍ये या संपर्काचा दुवा रद्द करायचा?"</string>
     <string name="splitConfirmation_positive_button" msgid="9129409098807939699">"दुवा रद्द करा"</string>
-    <string name="splitConfirmationWithPendingChanges" msgid="7719062163511895696">"आपण आधीपासून केलेले बदल आपण सेव्ह करू आणि एकाधिक संपर्कांमध्‍ये या संपर्काचा दुवा रद्द करू इच्छिता?"</string>
+    <string name="splitConfirmationWithPendingChanges" msgid="7719062163511895696">"तुम्ही आधीपासून केलेले बदल तुम्ही सेव्ह करू आणि एकाधिक संपर्कांमध्‍ये या संपर्काचा दुवा रद्द करू इच्छिता?"</string>
     <string name="splitConfirmationWithPendingChanges_positive_button" msgid="9073444264887244032">"सेव्ह करा आणि दुवा रद्द करा"</string>
-    <string name="joinConfirmation" msgid="8262614843581924365">"आपण आधीपासून केलेले बदल आपण सेव्ह करू आणि निवडलेल्या संपर्काशी दुवा जोडू इच्छिता?"</string>
+    <string name="joinConfirmation" msgid="8262614843581924365">"तुम्ही आधीपासून केलेले बदल तुम्ही सेव्ह करू आणि निवडलेल्या संपर्काशी दुवा जोडू इच्छिता?"</string>
     <string name="joinConfirmation_positive_button" msgid="4573092849769149516">"सेव्ह करा आणि दुवा जोडा"</string>
     <string name="contacts_linking_progress_bar" msgid="2846494347384549277">"दुवा साधत आहे"</string>
     <string name="contacts_unlinking_progress_bar" msgid="5989310198163726929">"दुवा रद्द करत आहे"</string>
@@ -57,7 +57,7 @@
     <string name="menu_linkedContacts" msgid="400444389718855621">"दुवा जोडलेले संपर्क"</string>
     <string name="menu_save" msgid="1727844363591825909">"सेव्ह करा"</string>
     <string name="titleJoinContactDataWith" msgid="6825255752748313944">"संपर्कांचा दुवा साधा"</string>
-    <string name="blurbJoinContactDataWith" msgid="5864256698061641841">"आपण <xliff:g id="NAME">%s</xliff:g> सह ज्या संपर्काचा दुवा जोडू इच्छिता तो निवडा:"</string>
+    <string name="blurbJoinContactDataWith" msgid="5864256698061641841">"तुम्ही <xliff:g id="NAME">%s</xliff:g> सह ज्या संपर्काचा दुवा जोडू इच्छिता तो निवडा:"</string>
     <string name="separatorJoinAggregateSuggestions" msgid="2831414448851313345">"सूचित संपर्क"</string>
     <string name="separatorJoinAggregateAll" msgid="7939932265026181043">"सर्व संपर्क"</string>
     <string name="contactsJoinedNamedMessage" msgid="1299418263439579455">"<xliff:g id="NAME">%s</xliff:g> चा दुवा जोडला"</string>
@@ -82,7 +82,7 @@
     <string name="menu_set_ring_tone" msgid="8728345772068064946">"रिंगटोन सेट करा"</string>
     <string name="menu_redirect_calls_to_vm" msgid="83331924006712755">"व्हॉइसमेल मार्गस्थ करा"</string>
     <string name="menu_unredirect_calls_to_vm" msgid="8725542534701811737">"व्हॉइसमेलवर मार्गस्थ रद्द करा"</string>
-    <string name="readOnlyContactWarning" msgid="7514475664695089411">"हा संपर्क केवळ-वाचनीय आहे, तो हटवणे शक्य नाही, परंतु आपण तो लपवू शकता."</string>
+    <string name="readOnlyContactWarning" msgid="7514475664695089411">"हा संपर्क केवळ-वाचनीय आहे, तो हटवणे शक्य नाही, परंतु तुम्ही तो लपवू शकता."</string>
     <string name="readOnlyContactWarning_positive_button" msgid="8914797212167683673">"संपर्क लपवा"</string>
     <string name="readOnlyContactDeleteConfirmation" msgid="4717023505914239809">"या संंपर्कावरील केवळ-वाचनीय खाती लपविली जातील, हटविली जाणार नाहीत."</string>
     <string name="single_delete_confirmation" msgid="3106905986948679720">"हा संपर्क हटवायचा?"</string>
@@ -159,7 +159,7 @@
     <string name="cancel_button_content_description" msgid="1288652456274531846">"रद्द करा"</string>
     <string name="back_arrow_content_description" msgid="4355362760545735065">"परत"</string>
     <string name="action_menu_back_from_edit_select" msgid="6435476408621731420">"बंद करा"</string>
-    <string name="aggregation_suggestion_edit_dialog_message" msgid="6549585283910518095">"निवडलेले संपर्क संपादित करण्‍यावर स्‍विच करायचे? आपण आत्तापर्यंत प्रविष्‍ट केलेली माहिती कॉपी केली जाईल."</string>
+    <string name="aggregation_suggestion_edit_dialog_message" msgid="6549585283910518095">"निवडलेले संपर्क संपादित करण्‍यावर स्‍विच करायचे? तुम्ही आत्तापर्यंत प्रविष्‍ट केलेली माहिती कॉपी केली जाईल."</string>
     <string name="contact_directory_description" msgid="683398073603909119">"निर्देशिका <xliff:g id="TYPE">%1$s</xliff:g>"</string>
     <string name="activity_title_settings" msgid="5464130076132770781">"सेटिंग्ज"</string>
     <string name="menu_settings" msgid="377929915873428211">"सेटिंग्ज"</string>
@@ -228,7 +228,7 @@
     <string name="content_description_recent_call_type_outgoing" msgid="5156553338985232744">"केले जाणारे"</string>
     <string name="content_description_recent_call_type_missed" msgid="7371810920196048204">"सुटलेले"</string>
     <string name="content_description_recent_call" msgid="5183800406316723676">"अलीकडील कॉल. <xliff:g id="CALL_TYPE">%1$s</xliff:g>. <xliff:g id="PHONE_NUMBER">%2$s</xliff:g>. <xliff:g id="DATE">%3$s</xliff:g>. परत कॉल करण्यासाठी क्लिक करा"</string>
-    <string name="message_from_you_prefix" msgid="7180706529908434482">"आपण: <xliff:g id="SMS_BODY">%s</xliff:g>"</string>
+    <string name="message_from_you_prefix" msgid="7180706529908434482">"तुम्ही: <xliff:g id="SMS_BODY">%s</xliff:g>"</string>
     <string name="editor_more_fields" msgid="3630987771304393421">"आणखी फील्ड"</string>
     <string name="editor_change_photo_content_description" msgid="820879708069864340">"संपर्क फोटो बदला"</string>
     <string name="editor_add_photo_content_description" msgid="4194479313465418120">"संपर्क फोटो जोडा"</string>
@@ -240,7 +240,7 @@
     <string name="quickcontact_suggestion_cancel_button" msgid="8236954313106630862">"रद्द करा"</string>
     <string name="suggestion_card_this_contact_title" msgid="3039457405374454914">"हा संपर्क"</string>
     <string name="suggestion_card_duplicates_title" msgid="9107788743178980902">"संभाव्य डुप्लिकेट"</string>
-    <string name="suggestion_card_help_message" msgid="4474061044387181093">"हे संपर्क म्हणजे कदाचित एकच व्यक्ती असू शकते. आपण त्या सर्वांना एकल दुवा म्हणून एकत्र जोडू शकता."</string>
+    <string name="suggestion_card_help_message" msgid="4474061044387181093">"हे संपर्क म्हणजे कदाचित एकच व्यक्ती असू शकते. तुम्ही त्या सर्वांना एकल दुवा म्हणून एकत्र जोडू शकता."</string>
     <string name="editor_delete_view_description" msgid="8583095381562991959">"<xliff:g id="DATA_TYPE">%1$s </xliff:g><xliff:g id="DATA_KIND">%2$s</xliff:g> हटवा"</string>
     <string name="editor_delete_view_description_short" msgid="7335518371270844912">"<xliff:g id="DATA_KIND">%s</xliff:g> हटवा"</string>
     <string name="menu_assistant" msgid="5760693735722052454">"सूचना"</string>
@@ -250,12 +250,12 @@
     <string name="navigation_drawer_label" msgid="4154758025620107419">"<xliff:g id="LABEL_NAME">%s</xliff:g> लेबल"</string>
     <string name="menu_title_groups" msgid="8356921831150278868">"लेबले"</string>
     <string name="menu_title_filters" msgid="8210922220185114527">"खाती"</string>
-    <string name="permission_explanation_header" msgid="5739405825039695327">"येथे आपला इतिहास एकत्र पहा"</string>
+    <string name="permission_explanation_header" msgid="5739405825039695327">"येथे तुमचा इतिहास एकत्र पहा"</string>
     <string name="permission_explanation_subheader_calendar_and_SMS" msgid="5411144298889835768">"इव्हेंट आणि संदेश"</string>
     <string name="permission_explanation_subheader_calendar" msgid="8785323496211704613">"इव्हेंट"</string>
     <string name="permission_explanation_subheader_SMS" msgid="1904552086449525567">"संदेश"</string>
     <string name="hamburger_feature_highlight_header" msgid="5762679188565585072">"सूचना"</string>
-    <string name="hamburger_feature_highlight_body" msgid="4403398269286898733">"आपले संपर्क व्यवस्थापित आणि उपयुक्त ठेवा"</string>
+    <string name="hamburger_feature_highlight_body" msgid="4403398269286898733">"तुमचे संपर्क व्यवस्थापित आणि उपयुक्त ठेवा"</string>
     <string name="undo" msgid="1425165101664071422">"पूर्ववत करा"</string>
     <string name="call_custom" msgid="6385303130912713318">"<xliff:g id="CUSTOM_LABEL">%s</xliff:g> ला कॉल करा"</string>
     <string name="call_home" msgid="1990519474420545392">"निवासस्‍थानी कॉल करा"</string>
@@ -303,7 +303,7 @@
     <string name="sms_by_shortcut" msgid="7741770672976099517">"<xliff:g id="CONTACT_NAME">%s</xliff:g> (संदेश)"</string>
     <string name="description_video_call" msgid="7120921378651700947">"व्‍हिडिओ कॉल करा"</string>
     <string name="clearFrequentsConfirmation_title" msgid="766292372438450432">"वारंवार सपर्क साधलेले साफ करायचे?"</string>
-    <string name="clearFrequentsConfirmation" msgid="2270554975938265734">"आपण संपर्क आणि फोन अ‍ॅप्‍स मधील वारंवार संपर्क साधलेली सूची साफ कराल आणि ईमेल अ‍ॅप्‍सना सुरवातीपासून तुमची पत्ता प्राधान्‍ये जाणून घेण्‍याची सक्ती कराल."</string>
+    <string name="clearFrequentsConfirmation" msgid="2270554975938265734">"तुम्ही संपर्क आणि फोन अ‍ॅप्‍स मधील वारंवार संपर्क साधलेली सूची साफ कराल आणि ईमेल अ‍ॅप्‍सना सुरवातीपासून तुमची पत्ता प्राधान्‍ये जाणून घेण्‍याची सक्ती कराल."</string>
     <string name="clearFrequentsProgress_title" msgid="5157001637482794212">"वारंवार सपर्क साधलेले साफ करत आहे..."</string>
     <string name="status_available" msgid="5586870015822828392">"उपलब्ध"</string>
     <string name="status_away" msgid="1838861100379804730">"दूर आहे"</string>
@@ -413,15 +413,15 @@
     <string name="fail_reason_could_not_open_file" msgid="2067725459821997463">"\"<xliff:g id="FILE_NAME">%1$s</xliff:g>\" उघडू शकलो नाही: <xliff:g id="EXACT_REASON">%2$s</xliff:g>."</string>
     <string name="fail_reason_could_not_initialize_exporter" msgid="707260459259688510">"निर्यातकर्ता प्रारंभ करू शकला नाही: \"<xliff:g id="EXACT_REASON">%s</xliff:g>\"."</string>
     <string name="fail_reason_no_exportable_contact" msgid="8728506011371262065">"कोणताही निर्यात करण्‍यायोग्‍य संपर्क नाही."</string>
-    <string name="missing_required_permission" msgid="5865884842972833120">"आपण आवश्‍यक असलेली एक परवानगी अक्षम केली आहे."</string>
+    <string name="missing_required_permission" msgid="5865884842972833120">"तुम्ही आवश्‍यक असलेली एक परवानगी अक्षम केली आहे."</string>
     <string name="fail_reason_error_occurred_during_export" msgid="3018855323913649063">"निर्यात दरम्‍यान एरर आलीt: \"<xliff:g id="EXACT_REASON">%s</xliff:g>\"."</string>
     <string name="fail_reason_io_error" msgid="6748358842976073255">"I/O एरर"</string>
     <string name="fail_reason_low_memory_during_import" msgid="875222757734882898">"पुरेशी मेमरी नाही. फाईल कदाचित खूप मोठी असू शकते."</string>
     <string name="fail_reason_not_supported" msgid="8219562769267148825">"स्वरूपन समर्थित नाही."</string>
     <string name="exporting_vcard_finished_title" msgid="4767045779458185251">"<xliff:g id="FILENAME">%s</xliff:g> निर्यात करणे समाप्त झाले."</string>
     <string name="exporting_vcard_finished_title_fallback" msgid="6060472638008218274">"संपर्क आयात करणे समाप्त झाले"</string>
-    <string name="exporting_vcard_finished_toast" msgid="1739055986856453882">"संपर्क निर्यात करणे समाप्त झाले, संपर्क सामायिक करण्यासाठी सूचनेवर क्लिक करा."</string>
-    <string name="touch_to_share_contacts" msgid="4882485525268469736">"संपर्क सामायिक करण्यासाठी टॅप करा."</string>
+    <string name="exporting_vcard_finished_toast" msgid="1739055986856453882">"संपर्क निर्यात करणे समाप्त झाले, संपर्क शेअर करण्यासाठी सूचनेवर क्लिक करा."</string>
+    <string name="touch_to_share_contacts" msgid="4882485525268469736">"संपर्क शेअर करण्यासाठी टॅप करा."</string>
     <string name="exporting_vcard_canceled_title" msgid="2652222370493306887">"<xliff:g id="FILENAME">%s</xliff:g> निर्यात करणे रद्द केले."</string>
     <string name="exporting_contact_list_title" msgid="9072240631534457415">"संपर्क डेटा निर्यात करत आहे"</string>
     <string name="exporting_contact_list_message" msgid="3367949209642931952">"संपर्क डेटा निर्यात केला जात आहे."</string>
@@ -458,15 +458,15 @@
     <string name="settings_accounts" msgid="350219740670774576">"खाती"</string>
     <string name="default_editor_account" msgid="699591683362420991">"नवीन संपर्कांसाठी डीफॉल्ट खाते"</string>
     <string name="settings_my_info_title" msgid="1534272456405343119">"माझी माहिती"</string>
-    <string name="set_up_profile" msgid="7370213843590143771">"आपले प्रोफाईल सेट करा"</string>
+    <string name="set_up_profile" msgid="7370213843590143771">"तुमचे प्रोफाईल सेट करा"</string>
     <string name="setting_about" msgid="7014388749752042863">"संपर्कांबद्दल"</string>
     <string name="share_favorite_contacts" msgid="4280926751003081042">"आवडते संपर्क शेअर करा"</string>
     <string name="share_contacts" msgid="8109287987498711664">"सर्व संपर्क शेअर करा"</string>
-    <string name="share_contacts_failure" msgid="1216431977330560559">"संपर्क सामायिक करण्यात अयशस्वी झाले."</string>
+    <string name="share_contacts_failure" msgid="1216431977330560559">"संपर्क शेअर करण्यात अयशस्वी झाले."</string>
     <string name="dialog_export" msgid="1628001095187741417">"संपर्क निर्यात करा"</string>
     <string name="dialog_import" msgid="2423592905927819672">"यावरून संपर्क आयात करा"</string>
-    <string name="share_error" msgid="948429331673358107">"हा संपर्क सामायिक केला जाऊ शकत नाही."</string>
-    <string name="no_contact_to_share" msgid="1276397530378323033">"सामायिक करण्यासाठी कोणतेही संपर्क नाहीत."</string>
+    <string name="share_error" msgid="948429331673358107">"हा संपर्क शेअर केला जाऊ शकत नाही."</string>
+    <string name="no_contact_to_share" msgid="1276397530378323033">"शेअर करण्यासाठी कोणतेही संपर्क नाहीत."</string>
     <string name="menu_contacts_filter" msgid="2165153460860262501">"प्रदर्शित करण्यासाठी संपर्क"</string>
     <string name="activity_title_contacts_filter" msgid="8275542497615516969">"प्रदर्शित करण्यासाठी संपर्क"</string>
     <string name="custom_list_filter" msgid="9048525797111024204">"दृश्य सानुकूल करा"</string>
@@ -519,7 +519,7 @@
     <string name="dismiss_sync_alert" msgid="8941131776605093596">"डिसमिस करा"</string>
     <string name="account_sync_off" msgid="7646143254066243924">"स्वयं-संकालन बंद आहे. चालू करण्यासाठी टॅप करा."</string>
     <string name="turn_auto_sync_on_dialog_title" msgid="333965456829670541">"स्वयं-संकालन चालू करायचे?"</string>
-    <string name="turn_auto_sync_on_dialog_body" msgid="5025336917533693997">"फक्त संपर्कांमध्ये नाही, तर आपण सर्व अॅप्स आणि खात्यांमध्ये करता ते बदल, वेब आणि आपल्या अन्य डिव्हाइसेस दरम्यान अपडेट ठेवले जातील."</string>
+    <string name="turn_auto_sync_on_dialog_body" msgid="5025336917533693997">"फक्त संपर्कांमध्ये नाही, तर तुम्ही सर्व अॅप्स आणि खात्यांमध्ये करता ते बदल, वेब आणि आपल्या अन्य डिव्हाइसेस दरम्यान अपडेट ठेवले जातील."</string>
     <string name="turn_auto_sync_on_dialog_confirm_btn" msgid="1070332014838295599">"चालू करा"</string>
     <string name="connection_error_message" msgid="7046408921529247089">"कोणतेही कनेक्शन नाही"</string>
     <string name="single_sim_display_label" msgid="6985770499439371497">"सिम"</string>
diff --git a/res/values-my/strings.xml b/res/values-my/strings.xml
index 1f82bc443..ec365f511 100644
--- a/res/values-my/strings.xml
+++ b/res/values-my/strings.xml
@@ -55,7 +55,7 @@
     <string name="contacts_unlinking_progress_bar" msgid="5989310198163726929">"လင့်ခ်ဖြုတ်နေသည်"</string>
     <string name="menu_joinAggregate" msgid="3599512127797513606">"လင့်ခ်"</string>
     <string name="menu_linkedContacts" msgid="400444389718855621">"လင့်ခ်အဆက်အသွယ်များ ကြည့်ရန်"</string>
-    <string name="menu_save" msgid="1727844363591825909">"သိမ်းဆည်းရန်"</string>
+    <string name="menu_save" msgid="1727844363591825909">"သိမ်းရန်"</string>
     <string name="titleJoinContactDataWith" msgid="6825255752748313944">"အဆက်အသွယ်များ ချိတ်ဆက်ရန်"</string>
     <string name="blurbJoinContactDataWith" msgid="5864256698061641841">"<xliff:g id="NAME">%s</xliff:g> နှင့်ချိတ်လိုသည့် အဆက်အသွယ်ကို ရွေးပါ −"</string>
     <string name="separatorJoinAggregateSuggestions" msgid="2831414448851313345">"အကြုံပြုလိုသော အဆက်အသွယ်များ"</string>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index c18fe8962..c11b04ac2 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -163,7 +163,7 @@
     <string name="contact_directory_description" msgid="683398073603909119">"Directory <xliff:g id="TYPE">%1$s</xliff:g>"</string>
     <string name="activity_title_settings" msgid="5464130076132770781">"Instellingen"</string>
     <string name="menu_settings" msgid="377929915873428211">"Instellingen"</string>
-    <string name="menu_help" msgid="1680178646764069976">"Help en feedback"</string>
+    <string name="menu_help" msgid="1680178646764069976">"Hulp en feedback"</string>
     <string name="organization_company_and_title" msgid="6718207751363732025">"<xliff:g id="COMPANY_0">%2$s</xliff:g>, <xliff:g id="COMPANY_1">%1$s</xliff:g>"</string>
     <string name="non_phone_caption" msgid="1541655052330027380">"Telefoonnummer"</string>
     <string name="non_phone_add_to_contacts" msgid="6590985286250471169">"Toevoegen aan contacten"</string>
@@ -227,7 +227,7 @@
     <string name="content_description_recent_call_type_incoming" msgid="5210739096863511410">"binnenkomend"</string>
     <string name="content_description_recent_call_type_outgoing" msgid="5156553338985232744">"uitgaand"</string>
     <string name="content_description_recent_call_type_missed" msgid="7371810920196048204">"gemist"</string>
-    <string name="content_description_recent_call" msgid="5183800406316723676">"recente oproep. <xliff:g id="CALL_TYPE">%1$s</xliff:g>. <xliff:g id="PHONE_NUMBER">%2$s</xliff:g>. <xliff:g id="DATE">%3$s</xliff:g>. klik om terug te bellen"</string>
+    <string name="content_description_recent_call" msgid="5183800406316723676">"recent gesprek. <xliff:g id="CALL_TYPE">%1$s</xliff:g>. <xliff:g id="PHONE_NUMBER">%2$s</xliff:g>. <xliff:g id="DATE">%3$s</xliff:g>. klik om terug te bellen"</string>
     <string name="message_from_you_prefix" msgid="7180706529908434482">"Jij: <xliff:g id="SMS_BODY">%s</xliff:g>"</string>
     <string name="editor_more_fields" msgid="3630987771304393421">"Meer velden"</string>
     <string name="editor_change_photo_content_description" msgid="820879708069864340">"Contactfoto wijzigen"</string>
@@ -481,9 +481,9 @@
     <string name="action_menu_back_from_search" msgid="8793348588949233348">"stoppen met zoeken"</string>
     <string name="description_clear_search" msgid="688023606766232904">"Zoekopdracht wissen"</string>
     <string name="select_account_dialog_title" msgid="5478489655696599219">"Account"</string>
-    <string name="set_default_account" msgid="4311613760725609801">"Altijd gebruiken voor oproepen"</string>
-    <string name="call_with_a_note" msgid="8389827628360791676">"Oproep met een notitie"</string>
-    <string name="call_subject_hint" msgid="3043028982108363572">"Typ een notitie om te verzenden met de oproep…"</string>
+    <string name="set_default_account" msgid="4311613760725609801">"Altijd gebruiken voor gesprekken"</string>
+    <string name="call_with_a_note" msgid="8389827628360791676">"Gesprek met een notitie"</string>
+    <string name="call_subject_hint" msgid="3043028982108363572">"Typ een notitie om te verzenden met het gesprek…"</string>
     <string name="send_and_call_button" msgid="7740295432834590737">"VERZENDEN EN BELLEN"</string>
     <string name="call_subject_limit" msgid="4545212901205397669">"<xliff:g id="COUNT">%1$s</xliff:g>/<xliff:g id="LIMIT">%2$s</xliff:g>"</string>
     <string name="call_subject_type_and_number" msgid="7667188212129152558">"<xliff:g id="TYPE">%1$s</xliff:g> • <xliff:g id="NUMBER">%2$s</xliff:g>"</string>
diff --git a/res/values-pl/strings.xml b/res/values-pl/strings.xml
index 82fe96572..fcd1521ba 100644
--- a/res/values-pl/strings.xml
+++ b/res/values-pl/strings.xml
@@ -446,7 +446,7 @@
     <string name="reading_vcard_failed_title" msgid="4251647443358422855">"Nie można odczytać danych vCard"</string>
     <string name="importing_vcard_finished_title" msgid="3341541727268747967">"Zakończono importowanie pliku vCard <xliff:g id="FILENAME">%s</xliff:g>"</string>
     <string name="importing_vcard_canceled_title" msgid="2147475978165599336">"Anulowano importowanie pliku <xliff:g id="FILENAME">%s</xliff:g>"</string>
-    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"Plik <xliff:g id="FILENAME">%s</xliff:g> zostanie za chwilę zaimportowany."</string>
+    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"Plik „<xliff:g id="FILENAME">%s</xliff:g>” zostanie za chwilę zaimportowany."</string>
     <string name="vcard_import_will_start_message_with_default_name" msgid="1022969530654129470">"Plik zostanie za chwilę zaimportowany."</string>
     <string name="vcard_import_request_rejected_message" msgid="2890471184508516011">"Żądanie importu danych vCard zostało odrzucone. Spróbuj ponownie później."</string>
     <string name="contacts_export_will_start_message" msgid="8538705791417534431">"Kontakty zostaną wyeksportowane wkrótce."</string>
diff --git a/res/values-pt-rBR/strings.xml b/res/values-pt-rBR/strings.xml
index f0b10f3dd..523158735 100644
--- a/res/values-pt-rBR/strings.xml
+++ b/res/values-pt-rBR/strings.xml
@@ -393,7 +393,7 @@
     <string name="savingDisplayGroups" msgid="2133152192716475939">"Salvando opções de exibição…"</string>
     <string name="listCustomView" msgid="1840624396582117590">"Visualização personalizada"</string>
     <string name="dialog_new_contact_account" msgid="4969619718062454756">"Salvar contatos importados em:"</string>
-    <string name="import_from_sim" msgid="6912071635295799131">"Cartão SIM"</string>
+    <string name="import_from_sim" msgid="6912071635295799131">"Chip"</string>
     <string name="import_from_sim_summary_fmt" msgid="5169032449686348118">"SIM <xliff:g id="SIM_NAME">%1$s</xliff:g>"</string>
     <plurals name="import_from_sim_secondary_contact_count_fmt" formatted="false" msgid="5821095773211149295">
       <item quantity="one">%1$d contato</item>
@@ -506,14 +506,14 @@
     <string name="dynamic_shortcut_contact_removed_message" msgid="6845645831837615899">"O contato foi removido"</string>
     <string name="sim_import_button_text" msgid="4270841592496619317">"Importar"</string>
     <string name="sim_import_title_none_selected" msgid="6910517331401970693">"Selecionar contatos"</string>
-    <string name="sim_import_empty_message" msgid="7743815244380189651">"Não há contatos no seu cartão SIM"</string>
+    <string name="sim_import_empty_message" msgid="7743815244380189651">"Não há contatos no seu chip"</string>
     <string name="sim_import_contact_exists_toast" msgid="1503743663717316732">"O contato já existe na sua lista"</string>
     <plurals name="sim_import_success_toast_fmt" formatted="false" msgid="8572156521110906443">
-      <item quantity="one"><xliff:g id="COUNT">%d</xliff:g> contato importado do SIM</item>
-      <item quantity="other"><xliff:g id="COUNT">%d</xliff:g> contatos importados do SIM</item>
+      <item quantity="one"><xliff:g id="COUNT">%d</xliff:g> contato importado do chip</item>
+      <item quantity="other"><xliff:g id="COUNT">%d</xliff:g> contatos importados do chip</item>
     </plurals>
-    <string name="sim_import_failed_toast" msgid="5559267299793622705">"Falha ao importar contatos do SIM"</string>
-    <string name="sim_import_title" msgid="2511154832820812822">"Importar do SIM"</string>
+    <string name="sim_import_failed_toast" msgid="5559267299793622705">"Falha ao importar contatos do chip"</string>
+    <string name="sim_import_title" msgid="2511154832820812822">"Importar do chip"</string>
     <string name="sim_import_cancel_content_description" msgid="1619832410253424452">"Cancelar"</string>
     <string name="auto_sync_off" msgid="2180147284456026587">"A sincronização automática está desativada. Toque para ativar."</string>
     <string name="dismiss_sync_alert" msgid="8941131776605093596">"Dispensar"</string>
@@ -524,10 +524,10 @@
     <string name="connection_error_message" msgid="7046408921529247089">"Sem conexão"</string>
     <string name="single_sim_display_label" msgid="6985770499439371497">"SIM"</string>
     <string name="show_more_content_description" msgid="4277410425912984722">"Mostrar mais"</string>
-    <string name="importing_sim_finished_title" msgid="5196369441294050721">"A importação do cartão SIM foi concluída"</string>
+    <string name="importing_sim_finished_title" msgid="5196369441294050721">"A importação do chip foi concluída"</string>
     <string name="importing_sim_failed_title" msgid="39706901030537985">"Falha na importação"</string>
-    <string name="importing_sim_failed_message" msgid="3345258302998021066">"Não foi possível importar os contatos do cartão SIM"</string>
-    <string name="importing_sim_in_progress_title" msgid="3638299581276676109">"Importando do cartão SIM"</string>
+    <string name="importing_sim_failed_message" msgid="3345258302998021066">"Não foi possível importar os contatos do chip"</string>
+    <string name="importing_sim_in_progress_title" msgid="3638299581276676109">"Importando do chip"</string>
     <string name="contacts_default_notification_channel" msgid="4754058700611188581">"Notificações"</string>
     <string name="yes_button" msgid="1268479086848288060">"Sim"</string>
     <string name="no_button" msgid="5742815694687835125">"Não"</string>
diff --git a/res/values-pt-rPT/strings.xml b/res/values-pt-rPT/strings.xml
index 1412a9ac8..c7b1881ae 100644
--- a/res/values-pt-rPT/strings.xml
+++ b/res/values-pt-rPT/strings.xml
@@ -345,7 +345,7 @@
     <string name="label_notes" msgid="8337354953278341042">"Notas"</string>
     <string name="label_custom_field" msgid="1994056912242214426">"Personalizado"</string>
     <string name="label_sip_address" msgid="7252153678613978127">"SIP"</string>
-    <string name="websiteLabelsGroup" msgid="4202998982804009261">"Web site"</string>
+    <string name="websiteLabelsGroup" msgid="4202998982804009261">"Website"</string>
     <string name="groupsLabel" msgid="7000816729542098972">"Etiquetas"</string>
     <string name="email_home" msgid="8573740658148184279">"Enviar email para residência"</string>
     <string name="email_mobile" msgid="2042889209787989814">"Enviar email para telemóvel"</string>
diff --git a/res/values-pt/strings.xml b/res/values-pt/strings.xml
index f0b10f3dd..523158735 100644
--- a/res/values-pt/strings.xml
+++ b/res/values-pt/strings.xml
@@ -393,7 +393,7 @@
     <string name="savingDisplayGroups" msgid="2133152192716475939">"Salvando opções de exibição…"</string>
     <string name="listCustomView" msgid="1840624396582117590">"Visualização personalizada"</string>
     <string name="dialog_new_contact_account" msgid="4969619718062454756">"Salvar contatos importados em:"</string>
-    <string name="import_from_sim" msgid="6912071635295799131">"Cartão SIM"</string>
+    <string name="import_from_sim" msgid="6912071635295799131">"Chip"</string>
     <string name="import_from_sim_summary_fmt" msgid="5169032449686348118">"SIM <xliff:g id="SIM_NAME">%1$s</xliff:g>"</string>
     <plurals name="import_from_sim_secondary_contact_count_fmt" formatted="false" msgid="5821095773211149295">
       <item quantity="one">%1$d contato</item>
@@ -506,14 +506,14 @@
     <string name="dynamic_shortcut_contact_removed_message" msgid="6845645831837615899">"O contato foi removido"</string>
     <string name="sim_import_button_text" msgid="4270841592496619317">"Importar"</string>
     <string name="sim_import_title_none_selected" msgid="6910517331401970693">"Selecionar contatos"</string>
-    <string name="sim_import_empty_message" msgid="7743815244380189651">"Não há contatos no seu cartão SIM"</string>
+    <string name="sim_import_empty_message" msgid="7743815244380189651">"Não há contatos no seu chip"</string>
     <string name="sim_import_contact_exists_toast" msgid="1503743663717316732">"O contato já existe na sua lista"</string>
     <plurals name="sim_import_success_toast_fmt" formatted="false" msgid="8572156521110906443">
-      <item quantity="one"><xliff:g id="COUNT">%d</xliff:g> contato importado do SIM</item>
-      <item quantity="other"><xliff:g id="COUNT">%d</xliff:g> contatos importados do SIM</item>
+      <item quantity="one"><xliff:g id="COUNT">%d</xliff:g> contato importado do chip</item>
+      <item quantity="other"><xliff:g id="COUNT">%d</xliff:g> contatos importados do chip</item>
     </plurals>
-    <string name="sim_import_failed_toast" msgid="5559267299793622705">"Falha ao importar contatos do SIM"</string>
-    <string name="sim_import_title" msgid="2511154832820812822">"Importar do SIM"</string>
+    <string name="sim_import_failed_toast" msgid="5559267299793622705">"Falha ao importar contatos do chip"</string>
+    <string name="sim_import_title" msgid="2511154832820812822">"Importar do chip"</string>
     <string name="sim_import_cancel_content_description" msgid="1619832410253424452">"Cancelar"</string>
     <string name="auto_sync_off" msgid="2180147284456026587">"A sincronização automática está desativada. Toque para ativar."</string>
     <string name="dismiss_sync_alert" msgid="8941131776605093596">"Dispensar"</string>
@@ -524,10 +524,10 @@
     <string name="connection_error_message" msgid="7046408921529247089">"Sem conexão"</string>
     <string name="single_sim_display_label" msgid="6985770499439371497">"SIM"</string>
     <string name="show_more_content_description" msgid="4277410425912984722">"Mostrar mais"</string>
-    <string name="importing_sim_finished_title" msgid="5196369441294050721">"A importação do cartão SIM foi concluída"</string>
+    <string name="importing_sim_finished_title" msgid="5196369441294050721">"A importação do chip foi concluída"</string>
     <string name="importing_sim_failed_title" msgid="39706901030537985">"Falha na importação"</string>
-    <string name="importing_sim_failed_message" msgid="3345258302998021066">"Não foi possível importar os contatos do cartão SIM"</string>
-    <string name="importing_sim_in_progress_title" msgid="3638299581276676109">"Importando do cartão SIM"</string>
+    <string name="importing_sim_failed_message" msgid="3345258302998021066">"Não foi possível importar os contatos do chip"</string>
+    <string name="importing_sim_in_progress_title" msgid="3638299581276676109">"Importando do chip"</string>
     <string name="contacts_default_notification_channel" msgid="4754058700611188581">"Notificações"</string>
     <string name="yes_button" msgid="1268479086848288060">"Sim"</string>
     <string name="no_button" msgid="5742815694687835125">"Não"</string>
diff --git a/res/values-ru/strings.xml b/res/values-ru/strings.xml
index 3676dcce8..d72da36b3 100644
--- a/res/values-ru/strings.xml
+++ b/res/values-ru/strings.xml
@@ -446,7 +446,7 @@
     <string name="reading_vcard_failed_title" msgid="4251647443358422855">"Ошибка чтения vCard-файла"</string>
     <string name="importing_vcard_finished_title" msgid="3341541727268747967">"Импорт файла vCard <xliff:g id="FILENAME">%s</xliff:g> завершен"</string>
     <string name="importing_vcard_canceled_title" msgid="2147475978165599336">"Импорт файла \"<xliff:g id="FILENAME">%s</xliff:g>\" отменен"</string>
-    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"Импорт <xliff:g id="FILENAME">%s</xliff:g> начнется в ближайшее время."</string>
+    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"Импорт скоро начнется (<xliff:g id="FILENAME">%s</xliff:g>)"</string>
     <string name="vcard_import_will_start_message_with_default_name" msgid="1022969530654129470">"Файл будет импортирован в ближайшее время."</string>
     <string name="vcard_import_request_rejected_message" msgid="2890471184508516011">"Запрос на импорт данных vCard отклонен. Повторите попытку позже."</string>
     <string name="contacts_export_will_start_message" msgid="8538705791417534431">"Контакты скоро будут экспортированы"</string>
diff --git a/res/values-th/strings.xml b/res/values-th/strings.xml
index 70c1017b0..14f91cbbe 100644
--- a/res/values-th/strings.xml
+++ b/res/values-th/strings.xml
@@ -434,7 +434,7 @@
     <string name="reading_vcard_failed_title" msgid="4251647443358422855">"ไม่สามารถอ่านข้อมูล vCard"</string>
     <string name="importing_vcard_finished_title" msgid="3341541727268747967">"นำเข้า vCard <xliff:g id="FILENAME">%s</xliff:g> เรียบร้อยแล้ว"</string>
     <string name="importing_vcard_canceled_title" msgid="2147475978165599336">"ยกเลิกการนำเข้า <xliff:g id="FILENAME">%s</xliff:g> แล้ว"</string>
-    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"การนำเข้า <xliff:g id="FILENAME">%s</xliff:g> จะเกิดขึ้นในไม่ช้า"</string>
+    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"ระบบจะนำเข้า <xliff:g id="FILENAME">%s</xliff:g> เร็วๆ นี้"</string>
     <string name="vcard_import_will_start_message_with_default_name" msgid="1022969530654129470">"ไฟล์จะถููกนำเข้าในไม่ช้า"</string>
     <string name="vcard_import_request_rejected_message" msgid="2890471184508516011">"คำขอนำเข้า vCard ถูกปฏิเสธ ลองใหม่ภายหลัง"</string>
     <string name="contacts_export_will_start_message" msgid="8538705791417534431">"ระบบจะส่งออกรายชื่อติดต่อในไม่ช้า"</string>
diff --git a/res/values-uz/strings.xml b/res/values-uz/strings.xml
index 5600efd7e..f8d5c0004 100644
--- a/res/values-uz/strings.xml
+++ b/res/values-uz/strings.xml
@@ -489,7 +489,7 @@
     <string name="call_subject_type_and_number" msgid="7667188212129152558">"<xliff:g id="TYPE">%1$s</xliff:g> • <xliff:g id="NUMBER">%2$s</xliff:g>"</string>
     <string name="about_build_version" msgid="1765533099416999801">"Versiyasi"</string>
     <string name="about_open_source_licenses" msgid="6479990452352919641">"Ochiq kodli DT litsenziyalari"</string>
-    <string name="about_open_source_licenses_summary" msgid="57418386931763994">"Ochiq kodli DT litsenziyalari haqida ma’lumot"</string>
+    <string name="about_open_source_licenses_summary" msgid="57418386931763994">"Ochiq kodli DT litsenziyalari haqida axborot"</string>
     <string name="about_privacy_policy" msgid="3705518622499152626">"Maxfiylik siyosati"</string>
     <string name="about_terms_of_service" msgid="4642400812150296723">"Foydalanish shartlari"</string>
     <string name="activity_title_licenses" msgid="5467767062737708066">"Ochiq kodli DT litsenziyalari"</string>
diff --git a/res/values-zh-rCN/strings.xml b/res/values-zh-rCN/strings.xml
index 2da207cda..4ec9ac2da 100644
--- a/res/values-zh-rCN/strings.xml
+++ b/res/values-zh-rCN/strings.xml
@@ -434,7 +434,7 @@
     <string name="reading_vcard_failed_title" msgid="4251647443358422855">"无法读取 vCard 数据"</string>
     <string name="importing_vcard_finished_title" msgid="3341541727268747967">"导入 vCard <xliff:g id="FILENAME">%s</xliff:g> 已完成"</string>
     <string name="importing_vcard_canceled_title" msgid="2147475978165599336">"已取消导入 <xliff:g id="FILENAME">%s</xliff:g>"</string>
-    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"<xliff:g id="FILENAME">%s</xliff:g> 将在稍后导入。"</string>
+    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"<xliff:g id="FILENAME">%s</xliff:g>将在稍后导入。"</string>
     <string name="vcard_import_will_start_message_with_default_name" msgid="1022969530654129470">"系统稍后就会导入该文件。"</string>
     <string name="vcard_import_request_rejected_message" msgid="2890471184508516011">"vCard 导入请求遭拒，请稍后重试。"</string>
     <string name="contacts_export_will_start_message" msgid="8538705791417534431">"即将导出联系人。"</string>
diff --git a/res/values-zh-rHK/strings.xml b/res/values-zh-rHK/strings.xml
index b8a86bb05..1b3010e1d 100644
--- a/res/values-zh-rHK/strings.xml
+++ b/res/values-zh-rHK/strings.xml
@@ -434,7 +434,7 @@
     <string name="reading_vcard_failed_title" msgid="4251647443358422855">"無法讀取 vCard 資料"</string>
     <string name="importing_vcard_finished_title" msgid="3341541727268747967">"已完成匯入 vCard <xliff:g id="FILENAME">%s</xliff:g>"</string>
     <string name="importing_vcard_canceled_title" msgid="2147475978165599336">"已取消匯入「<xliff:g id="FILENAME">%s</xliff:g>」的操作"</string>
-    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"<xliff:g id="FILENAME">%s</xliff:g> 將在稍後匯入。"</string>
+    <string name="vcard_import_will_start_message" msgid="2804911199145873396">"<xliff:g id="FILENAME">%s</xliff:g>將在稍後匯入。"</string>
     <string name="vcard_import_will_start_message_with_default_name" msgid="1022969530654129470">"稍後即將匯入檔案。"</string>
     <string name="vcard_import_request_rejected_message" msgid="2890471184508516011">"vCard 匯入要求已被拒，請稍後再試。"</string>
     <string name="contacts_export_will_start_message" msgid="8538705791417534431">"系統即將匯出聯絡人。"</string>
-- 
2.17.1

